sunpy/net/jsoc/jsoc.py

```diff
@@ -90,9 +90,21 @@ def append(self, table):
     def response_block_properties(self):
         """
         Returns a set of class attributes on all the response blocks.
+
+        Returns
+        -------
+        s : `set`
+            List of strings, containing attribute names in the response blocks.
         """
-        warnings.warn("The JSOC client does not support response block properties.", SunpyUserWarning)
-        return set()
+        s = {a if not a.startswith('_') else None for a in dir(self[0])}
+        for resp in self[1:]:
+            if len(s) == 0:
+                break
+            s = s.intersection({a if not a.startswith('_') else None for a in dir(resp)})
+
+        if None in s:
+            s.remove(None)
+        return s
```

Comment:
This will need a test.

Reply:
VSO one also does not have any test for this.

Reply:
Then one should be added for both.